### PR TITLE
Verifier sub classes

### PR DESCRIPTION
**Pull request by zamattiac**
Equivalent of page sub classes

### CODE REVIEW

File: oop_verify.py [hunk 1]
```diff
@@ -82,3 +102,211 @@ def spot_check(self):
...
                     self.pages.pop(page)
         return
+
+
+class ProjectDashboardVerifier(Verifier):
+    def __init__(self):
+        Verifier.__init__(self)
+        self.pages = []
+        self.minimum_size = 410
+        self.page_elements = [
```
Comment (sf2ne) [inline oop_verify.py:112]: If page_elements is the same for all the subclasses (or for most subclasses) can it just be made a field of the superclass? And then altered in the subclasses if needed.

**\* Updated in new commit

File: oop_verify.py [hunk 2]
```diff
@@ -124,14 +124,9 @@ class ProjectFilesVerifier(Verifier):
     def __init__(self):
         Verifier.__init__(self)
         self.pages = []
-        self.minimum_size = 410
+        self.minimum_size = 380
         self.page_elements = [
```
Comment (sf2ne) [inline oop_verify.py:128]: Are you not gunna add these elements to self.page_elements in the Verifier super class? Do we not need them?

File: oop_verify.py [hunk 3]
```diff
@@ -124,14 +124,9 @@ class ProjectFilesVerifier(Verifier):
...
-            '#nodeTitleEditable',                                # Title
```
Comment (sf2ne) [inline oop_verify.py]: By "these elements" I meant the specific elements you took out of page_elements in all the subclasses

File: oop_verify.py (hunk 2 above)
Comment (zamattiac) [inline oop_verify.py:128]: @sf2ne Are you referring to the size? Page elements refers to a list of actual HTML elements used for spot check.

File: oop_verify.py (hunk 3 above)
Comment (zamattiac) [inline oop_verify.py]: @sf2ne All those elements were copy-pasted along with the project dashboard subclass to lay out all the other subclasses. They don't appear on other pages, except registration dashboard.

**Pull request merged**